### DISCUSSION TITLE
Is there a way to add plugin to stylelint?

### DISCUSSION THREAD

**Discussion by exSnake** [super-linter/super-linter#4395] [Q&A]
In .stylelintrc.json i have 
```
"plugins": [
    "stylelint-selector-bem-pattern"
  ]
```
but the action need to install an npm package before running, is it possible to do that?

**Comment by ferrarimarco** [comment]
There is currently no way of doing that, unless we install that package in the container image. You can surely build your own container image based on super-linter, although you'll need to maintain it.

Can you please open a new feature request for this? Thanks!
